node/examples/example1.js

```diff
@@ -45,19 +49,33 @@ var listening = ready(function (err) {
         throw err;
     }
 
+    client.makeSubChannel({
+        serviceName: 'server',
+        peers: [server.hostPort]
+    });
+
     client
-        .request({host: '127.0.0.1:4040'})
+        .request({serviceName: 'server'})
         .send('func1', "arg 1", "arg 2", function (err, res) {
+            assert.ifError(err);
+            assert.equal(res.ok, true);
             console.log('normal res: ' + res.arg2.toString() + ' ' + res.arg3.toString());
+            done();
         });
     client
-        .request({host: '127.0.0.1:4040'})
+        .request({serviceName: 'server'})
         .send('func2', "arg 1", "arg 2", function (err, res) {
+            assert.ifError(err);
```

Comment:
Let’s try not to replicate this pattern anymore. Do `if (err) return assert.end(err)`.